python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -1207,3 +1207,47 @@ async def __aenter__(self: TRemoteGroup) -> TRemoteGroup:
 
     async def __aexit__(self, *args: typing.Any) -> None:
         await self.close()
+
+    async def show_auth_status(self) -> None:
+        """Shows the authorization status of a group"""
+
+        async def _log_auth_status(c: salobj.Remote) -> None:
```

Comment:
I think having this method defined inside the other methods might be a bit of an anti-pattern. You could use `self._aget_topic_samples_for_components` defined [here](https://github.com/lsst-ts/ts_observatory_control/blob/7c64bbd5c31761ad45d168250ca122b85601f281/python/lsst/ts/observatory/control/remote_group.py#L981) to get the values of the auth list events and then log them here.